containers/src/Data/Map/Internal.hs

```diff
@@ -2467,10 +2467,8 @@ filterMissing f = WhenMissing
 -- | Filter the entries whose keys are missing from the other map
 -- using some 'Applicative' action.
 --
--- @
--- filterAMissing f = Merge.Lazy.traverseMaybeMissing $
---   \k x -> (\b -> guard b *> Just x) <$> f k x
--- @
+-- > filterAMissing f = Merge.Lazy.traverseMaybeMissing $
+-- >   \k x -> (\b -> guard b *> Just x) <$> f k x
```

Comment:
This is a much more radical change. Why?

Reply:
It's cleaner in source code without the escaping, in my opinion. `>` is like `@` but takes the string literally.
The other instances cannot use `>` because they have markup linking to other parts of the documentation.
I could switch this one to `@` if you prefer.

Reply:
Don't we want `Merge.Lazy.traverseMaybeMissing` to be a link?

Reply:
We might, but it would be a little inconsistent since all [adjacent codes](https://hackage.haskell.org/package/containers-0.6.5.1/docs/Data-Map-Internal.html#v:filterMissing) and the corresponding functions [in IntMap](https://hackage.haskell.org/package/containers-0.6.6/docs/Data-IntMap-Internal.html#v:filterAMissing) all don't link to the functions they use.